js/main.js

```diff
@@ -0,0 +1,115 @@
+'use strict';
+(function () {
+  var OFFER_NUMBER = 8;
```

Comment:
```suggestion
  var OFFERS_NUMBER = 8;
```
по-хорошему множественное число, тк "предложения" - это множественное число, надо и по-английски также

Reply:
там еще ниже встречается тоже